state/state.go

```diff
@@ -82,6 +82,7 @@ type CollectorConfig struct {
 	DbHasSslkey             bool
 	DbExtraNames            []string
 	DbAllNames              bool
+	DbURL                   string
```

Comment:
It would be good to use `DbURLRedacted` (or similar) as the name, so its clear that this is not the same as the original config field.